src/app/static/static-values.ts

```diff
@@ -12,6 +14,22 @@ export class StaticValues {
   public static farmTotalSupply = 0.0;
   public static farmUsers = 0;
 
+  // Networks
+  public static NETWORK_ETH: Network = {
+    blockExplorerUrl: 'https://www.bscscan.com',
+    chainId: 1,
+    currencySymbol: 'ETH',
+    name: 'Ethereum Mainnet',
```

Comment:
add a field `ethparserName` =  `eth` and `bsc` accordingly

Reply:
I went with `chainName` (in my current working version) but i'll change it to `ethparserName`